packages/edit-site/src/components/global-styles/navigation-button.js

```diff
@@ -7,18 +7,29 @@ import {
 	FlexItem,
 	__experimentalHStack as HStack,
 } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
 import { Icon } from '@wordpress/icons';
 
 function NavigationButton( {
 	path,
 	icon,
 	children,
+	label: labelProp,
 	isBack = false,
 	...props
 } ) {
 	const navigator = useNavigator();
+
+	const defaultLabel = isBack
+		? __( 'Navigate to the previous screen' )
```

Comment:
Would "screen" be misleading here?

Reply:
You raise a valid point — although I'm not sure how else we could call what each `NavigatorScreen` component represents. I couldn't think of anything else than `screen` or `view`

Reply:
yeah, I think view would carry less assumptions.

Reply:
Done in [`af9a939` (#35325)](https://github.com/WordPress/gutenberg/pull/35325/commits/af9a939e21d652c7c90f8bccee63f3a4aec72cfd).

I've also updated this PR description and the related issue.